src/components/ContentWrapper.jsx

```diff
@@ -238,7 +257,7 @@ const ContentWrapper = (props) => {
   }) => {
     const noExperimentDisable = !experimentId ? disableIfNoExperiment : false;
     const pipelineStatusDisable = disabledByPipelineStatus && (
-      pipelineError || pipelineRunning || pipelineRunningError
+      backendError || pipelineRunning || pipelineRunningError
```

Comment:
is `backendError` going to be true if either `pipeline` or `gem2s` had an error?

Reply:
This is actually for showing if there are errors when getting (loading) the backends status. Renamed to `errorLoadingBackend` to make it clearer.